src/lib/application.ts

```diff
@@ -238,9 +238,23 @@ export class Application extends ChildableComponent<Application, AbstractCompone
      * @returns  The list of input files with expanded directories.
      */
     public expandInputFiles(inputFiles?: string[]): string[] {
-        let exclude: IMinimatch, files: string[] = [];
+        let exclude: Array<IMinimatch>, files: string[] = [];
         if (this.exclude) {
-            exclude = new Minimatch(this.exclude);
+            exclude = this.exclude.map(pattern => new Minimatch(pattern));
+        }
+
+        function isExcluded(fileName: string): boolean {
+            if (!exclude) {
```

Comment:
This should never happen, you're initialising it already.

Reply:
It was only initialized when this.exclude was set. I've updated the initializer a bit now to make it look prettier.